vms/volunteer/views.py

```diff
@@ -64,6 +65,23 @@ def delete_resume(request, volunteer_id):
     else:
         return HttpResponse(status=403)
 
+@login_required
+def get_job_list(request):
+    result = []
+    user = request.user
+    volunteer_id = user.volunteer.id
+    event_name = request.GET.get('event_name', None)
+    if not event_name:
+        return HttpResponse(result, content_type ="application/json")
+    event = get_event_by_name(event_name)
+    print(event)
+    event_jobs = get_jobs_by_event_id(event.id)
+    job_list = get_signed_up_jobs_for_volunteer(volunteer_id)
+    for event_job in event_jobs:
+        if(any(event_job.name == job for job in job_list)):

```

Comment:
`job_list` here contains only names of jobs so i cannot compare with `id`.

Reply:
so job_list is just a list of job names and not Job Objects ? Super bad then. Please write a new function which would return the job objects, and use for querying.

Reply:
@tonythomas01 ok i will try then.

Reply:
Do this only if you are changing the entire logic to use id instead. If not, since name is unique - we can even go with this.

Reply:
Ok so i think using name is fine since the same function that returned name is used at many places.